webextensions/api/alarms.json

```
@@ -6,7 +6,7 @@
           "mdn_url": "https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/alarms",
           "support": {
             "chrome": {
-              "version_added": "≤58"
+              "version_added": "21"
```

https://github.com/chromium/chromium/commit/47a047d26cd3a931148f59b2b035aa1deddc153c landed in [20.0.1131.0](https://github.com/chromium/chromium/releases/tag/20.0.1131.0) before [20.0.1131.0](https://github.com/chromium/chromium/releases/tag/20.0.1131.0), but dev-channel-only. https://github.com/chromium/chromium/commit/3f9911a53f606508015cffbfffea05bf4205a037 in [22.0.1209.0](https://github.com/chromium/chromium/releases/tag/22.0.1209.0) before 22.0.1229 enabled it in stable.

```suggestion
              "version_added": "22"
```